ci-operator/step-registry/cucushift/installer/check/nutanix/zones/cucushift-installer-check-nutanix-zones-commands.sh

```diff
@@ -13,17 +13,35 @@ declare prism_element3
 # shellcheck source=/dev/null
 source "${NUTANIX_AUTH_PATH}"
 
-node_zone_list=$(oc get nodes -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
-
 if [[ "$SINGLE_ZONE" == "true" ]]; then
+    node_zone_list=$(oc get nodes -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}')
     echo "Check all nodes running on single zone: $prism_element1"
     for zone in ${node_zone_list}; do
         if [ "$zone" != "$prism_element1" ]; then
             echo "Fail: fail to check single zone: $zone, expected: $prism_element1"
             exit 1
         fi
     done
+elif [[ "$COMPUTE_ZONE" != "" ]]; then
+    compute_pes=$(echo "$COMPUTE_ZONE" | sed -e "s/failure-domain-1/$prism_element1/g" -e "s/failure-domain-2/$prism_element2/g" -e "s/failure-domain-3/$prism_element3/g" | tr " " "\n" | sort -u | xargs)
+    compute_zone_list=$(oc get nodes -l node-role.kubernetes.io/worker= -o=jsonpath='{.items[*].metadata.labels.topology\.kubernetes\.io\/zone}' | tr " " "\n" | sort -u | xargs)
+    if [[ $compute_zone_list == "$compute_pes" ]]; then
+        echo "Pass: passed to check compute zone: $compute_zone_list, expected: $compute_pes"
+    else
+        echo "Fail: fail to check compute zone: $compute_zone_list, expected: $compute_pes"
+        exit 1
+    fi
+elif [[ "$CONTROL_PLANE_ZONE" != "" ]]; then
```

Comment:
Per the [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/54015/rehearse-54015-periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-nutanix-ipi-zones-f14/1810494905038934016/artifacts/nutanix-ipi-zones-f14/cucushift-installer-check-nutanix-zones/build-log.txt) , the script did not run this check.

Reply:
Fixed, thanks for catching it.